packages/web-components/src/keplr-connection/queryBankBalances.js

```diff
@@ -0,0 +1,20 @@
+import { QueryClient, createProtobufRpcClient } from '@cosmjs/stargate';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
+import { QueryClientImpl } from 'cosmjs-types/cosmos/bank/v1beta1/query';
+
+/**
+ * @param {string} address
+ * @param {string} rpc
+ */
+export const queryBankBalances = async (address, rpc) => {
```

Comment:
another copy of this? same [scaling consideration](https://github.com/Agoric/wallet-app/pull/51#discussion_r1088314116) applies. Filed:

 - https://github.com/Agoric/agoric-sdk/issues/6870